src/components/Accordian/_Accordian.scss

```diff
@@ -0,0 +1,88 @@
+.accordion {
+  position: relative;
+  display: block;
+  cursor: pointer;
+  transition: 0.4s;
+  border: 1px solid #232222;
+  background-color: #232222;
```

Comment:
The background color and font color of your Accordion do not match the colors in Figma.